wandb/sdk/artifacts/storage_policies/wandb_storage_policy.py

```diff
@@ -135,14 +135,18 @@ def load_file(
                 manifest_entry._download_url = None
         if manifest_entry._download_url is None:
             auth = None
-            if not _thread_local_api_settings.cookies:
-                assert self._api.api_key is not None
-                auth = ("api", self._api.api_key)
+            http_headers = _thread_local_api_settings.headers or {}
+            if self._api.access_token is not None:
+                auth_header = {"Authorization": "Bearer " + self._api.access_token}
+                http_headers.update(auth_header)
```

Comment:
It's not - but we are already configuring a new client here, and making an assumption that the user is supplying an api key, so I had to add the header here as well. I don't know why we are not using the internal api client. I think refactoring this would be outside the scope of this PR.